torch/_inductor/codegen/cpp.py

```diff
@@ -399,6 +399,21 @@ def _print_Abs(self, expr):
         assert len(expr.args) == 1
         return f"std::abs({self._print(expr.args[0])})"
 
+    def _print_Round(self, expr):
+        assert len(expr.args) == 1
+        return f"std::lrint({self._print(expr.args[0])})"
+
+    def _print_RoundDecimal(self, expr):
+        assert len(expr.args) == 2
+        number, ndigits = expr.args
+        assert isinstance(ndigits, sympy.Integer)
+
+        if number.is_integer and ndigits >= 0:
+            return self._print(number)
```

Comment:
This can be done inside the class `RoundDecimal` and we can remove all these ifs from all the codegen.